backend/src/entities/Edge.ts

```diff
@@ -5,7 +5,14 @@ import { Property } from './Property';
  * Represents an edge of a graph with additional information.
  */
 export interface Edge extends EdgeDescriptor {
+  /**
+   * labels-property of a neo4j-edge
```

Comment:
We should not make the connection to neo4j, since it can/will be used for other graph-dbs as well. This class is independent of the DB implementation.

Reply:
its not wrong, I just wanted to make sure that the interfaces are not created to fit neo4j, but are generic solutions that are independent of the graph-db.
ATM is okay like that, it's only a comment anyway.